src/app/components/Hero/Hero.jsx

```diff
@@ -3,16 +3,26 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Hero = ({ heroData }) => {
+  // Check if this list is one of "Staff Picks"
+  const isStaffPicksList = heroData.header === 'Staff Picks';
   const image = heroData.heroImageUrl;
+  // "Staff Picks" lists have section title
+  const sectionTitle = isStaffPicksList ?
+    <p className="hero-section-title">{heroData.sectionTitle}</p> : null;
+  const staffPicksHeroClassName = isStaffPicksList ? ' staff-picks-hero' : '';
+  // "Staff Picks" lists do not have a hero image but a full red background color
+  const heroImageDOM = isStaffPicksList ? null :
+    <div className="hero-image">
+      <img src={image} alt="" />
+    </div>;
```

Comment:
Would it be better to define the `sectionTitle`, `staffPicksHeroClassName` and `heroImageDom` using `let` and do one conditional test for isStaffPicksList and set the variables accordingly?

Rename `sectionTitle` to `sectionTitleDOM`

Reply:
sure!